en/lessons/gravity-model.md

```diff
@@ -0,0 +1,712 @@
+
+From a mathematical perspective, our gravity model is a type of [regression analysis](https://en.wikipedia.org/wiki/Regression_analysis), a means of comparing sets of variables in search of relationships between them. While not all gravity models use regression, the example in this tutorial does. This section covers in brief regression analyses, moving from a simple linear regression, to a multivariate linear regression, and finally to the negative binomial regression which is the basis of our model. Each builds upon the other.
+
+- simple linear regression
+- multivariate linear regression
+- negative binomial regression (our gravity model)
+
+## Simple Linear Regression
+
+The most basic regression analysis is a [simple linear regression](https://en.wikipedia.org/wiki/Simple_linear_regression) analysis. A simple linear regression of two variables (eg, county population, and number of vagrants) provides a way to quantify the relationship between those two variables. When you plot the values on a [scatter plot](https://en.wikipedia.org/wiki/Scatter_plot) (eg, county population on the x-axis, and number of vagrants on the y-axis), looking at the graph makes it clear that there is a loose but reasonably obvious linear relationship between them (Figure 6). Generally speaking, the greater the population, the more vagrants you find. The purpose of a simple linear regression is to calculate the formula that best represents the straight line that comes as close to as many of the points on the graph as possible. Since not all points fall directly on the line, but most are fairly close, Figure 6 suggests that population is a good, but not a perfect predictor of the number of vagrants from a given county.
+
+{% include figure.html filename="figure6.png" caption="Figure 6: A simple linear regression of county population (x-axis) and number of vagrants observed, 1777-1786 (y-axis). To make this graph more readable, Yorkshire has been excluded because of its very large population." %}
+
+There are many websites that provide calculator functions that will do this for you automatically, and proprietary software including Microsoft Excel and SPSS can also perform this calculation. The formula for a simple linear regression is:
+
+
+$$y = α + βx$$
+
+- "$$y$$" is a value on the y-axis (number of vagrants, in the example above).
```

Comment:
Sometimes when referring to equation components in the inline text, the LaTeX has been wrapped in `""` in this lesson, other times not. I think for consistency you should just not use `""` since the letters are already specially rendered in the math font once they're wrapped by `$$`.

Reply:
removed " " where it was used in this manner.